src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/UnicodeRanges.cs

```diff
@@ -32,9 +32,8 @@ private static UnicodeRange CreateEmptyRange(ref UnicodeRange range)
         {
             // If the range hasn't been created, create it now.
             // It's ok if two threads race and one overwrites the other's 'range' value.
-            var newRange = new UnicodeRange(0, 0);
-            Volatile.Write(ref range, newRange);
-            return newRange;
+            var range = new UnicodeRange(0, 0);
+            return range;
```

Comment:
> Also, not that the CLUI spec talks about this explicitly:

Yes, and it ends with this:
"It is explicitly *not* a requirement that a conforming implementation of the CLI guarantee that all
 state updates performed within a constructor be uniformly visible before the constructor
 completes. CIL generators can ensure this requirement themselves by inserting appropriate calls
 to the memory barrier or volatile write instructions."